### DISCUSSION TITLE
作者大大能不能给一下高清ddns-go的logo

### DISCUSSION THREAD

**Discussion by zxxhz** [jeessy2/ddns-go#768] [Q&A] [answered]
我已经把ddns-go上架了casaos的App Store。如果您不愿意的话我去联系删除。github上的ico放大很模糊，所以您能不能提供一下高清的logo。

**Comment by jeessy2** [comment, marked as answer]
没logo😓
感兴趣可以制作一个好看的logo

**Reply by zxxhz** [reply]
我连艺术细菌都没有😇

**Reply by lddsb** [reply]
那你ico的原图发一下呗 😄

**Reply by jeessy2** [reply]
建议用工具/AI 生成一个，有空在回传
